NEWS.md

```diff
@@ -11,11 +11,21 @@ Refer to the [Release Notes for
 v0.7](https://github.com/JuliaLang/julia/blob/master/HISTORY.md) for a
 detailed list of changes from Julia v0.6.
 
+Standard Library Changes
+------------------------
+
+* The `Libdl` module's methods `dlopen()` and `dlsym()` have been updated
+  to return `nothing` as the indicator of a nonexistent symbol or library.
```

Comment:
This gives the impression that the default behavior changed. Instead, it just gained a keyword.

Reply:
The behavior did change; if the library is not found, `nothing` is returned instead of `C_NULL`.

Reply:
Didn't it used to throw an error?

Reply:
Oh, you're right, I should be talking about `dlopen_e()` and `dlsym_e()` here.  Those are the ones that changed.

Reply:
Oh, actually, I missed that that happened. Those functions are exported and checking their output for NULL is their primary use case. I think we need to change that back and have the `_e` variants return `C_NULL` in the failure case.